installer/manifests/keptn/templates/_helpers.tpl

```diff
@@ -488,6 +489,7 @@ Usage:
 
 {{- define "keptn.initContainers.wait-for-keptn-mongo" -}}
 - name: "wait-for-keptn-mongo"
+  namespace: "{{ .Release.Namespace }}"
```

Comment:
this is not needed here since the init container is filled directly into the pod.spec.initConainers field